internal/providers/terraform/azure/testdata/image_test/image_test.golden

```diff
@@ -46,7 +46,7 @@
  ├─ storage_data_disk                                                                                                  
  │  └─ Storage (P4, LRS)                                                     1  months                           $5.81 
  └─ storage_data_disk                                                                                                  
-    └─ Storage (P1, LRS)                                                     1  months                           $0.78 
+    └─ Storage (P4, LRS)                                                     1  months                           $5.81 
```

Comment:
From what I can tell this is actually correct since the `disk_size_gb` is set to `null` so should default to 30GB and be a P4 disk.